Sources/NIO/ByteBuffer-conversions.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
```

Comment:
rename to `ByteBuffer+Conversions.swift`?

Reply:
I have no preference, but there's a precedence set by `ByteBuffer-aux` and `ByteBuffer-core`, etc

Reply:
no strong opinion either, its more of a swift pattern to use `+`

Reply:
@weissi @Lukasa

Reply:
I think we should use `+` in the future but for now I'd stick with existing conventions. Not strong at all

Reply:
Agreed